hz: 10

error_threshold: 0.5
time_threshold: 1.5
past_path_threshold: 50
person_num_limit: 30

# observation_noise_ratio: 0.05
observation_noise_ratio: 0.1
sigma_initial_P_theta: 6.2832
# sigma_initial_P_velocity: 3.0
sigma_initial_P_velocity: 5.0
# sigma_initial_P_omega: 3.1416
sigma_initial_P_omega: 6.2832

# sigma_Q_x: 3.0
# sigma_Q_y: 3.0
# sigma_Q_theta: 3.1416
# sigma_Q_velocity: 3.0
# sigma_Q_omega: 3.1416

sigma_Q_x: 1.5
sigma_Q_y: 1.5
sigma_Q_theta: 3.1416
sigma_Q_velocity: 3.0
# sigma_Q_omega: 3.1416
sigma_Q_omega: 6.2832

trajectory_z: 0.0
# data_num_th_visualize: 1
data_num_th_visualize: 3
data_num_th_mahalanovis: 6
predict_time: 2.0
predict_dt: 0.1
calc_future_trajectory_flag: false
visualize_future_trajectory_flag: false
visualize_past_trajectory_flag: true

# register_th: 1.5 #hog
register_th: 0.9 #mask r-cnn
# register_th: 0.8 #mobilenetssd
# duplicate_th: 0.35
duplicate_th: 0.5

target_frame: "map"
person_poses_topic_name: "/use_pixel/person_poses"

